### PR TITLE
doc: Update the model checking document

### PR DESCRIPTION
**Pull request by zpzigi754**
This PR updates the model-checking report. The results for realm_activate, rec_aux_count, rec_destroy, and version have been added.

The rendered version can be temporarily found [here](https://github.com/islet-project/islet/blob/update-mc-doc/doc/islet-model-checking.md).

### CODE REVIEW

File: doc/islet-model-checking.md [hunk 1]
```diff
@@ -370,8 +374,7 @@ Figure 4-1. Host Memory Modeling (Declaration)  | Figure 4-2. Host Memory Modeli
 
 ### Bugs Found
 
-We found one correctness bug in RMI features and one security bug in RMI granule
-undelegate during Islet's model checking.
+We found four correctness bug and one security bug during Islet's model checking.
```
Comment (bitboom) [inline doc/islet-model-checking.md:377]:
```suggestion
We found four correctness bugs and one security bug during Islet's model checking.
```

**Review by bitboom — APPROVED** [review]
Nice documentation !

File: doc/islet-model-checking.md (hunk 1 above)
Comment (zpzigi754) [reply doc/islet-model-checking.md:377]: Oh, thank you for the correction!

Comment (zpzigi754) [reply doc/islet-model-checking.md]: I corrected it with a separate commit, because the proxy environment prevents me from uploading images..

**Review by hihi-wang — APPROVED** [review]

**Review by jinbpark — APPROVED** [review]

**Pull request merged**